### DISCUSSION TITLE
Challenge not detected

### DISCUSSION THREAD

**Discussion by SadE54** [FlareSolverr/FlareSolverr#746] [Q&A]
Hi,

I'm trying to set up prowlarr + flaresolverr for a specific indexer that is using cloudfare.
In prowlarr if I'm trying to set up the indexer without flaresolverr, it cannot access to it (normal behavior)
I set up on a dedicated server my own flaresolverr instance , and i can ping it from a brower.
but then if I trying to add flaresolverr in my indexer settings , I cannot save because theres an unsknow error.
Looking at the info displayed on my flaresolverr server I have : 
https://gist.github.com/SadE54/d6c9b70c5793892d1a7f712f4b483862
It's saying 'challenge not detected!' 

How can i fix that ? 

Regards,

**Comment by SadE54** [comment]
I just added a full debug log here, with html content, as joined file .
[MobaXterm_192.168.1.51_20230327_083743.txt](https://github.com/FlareSolverr/FlareSolverr/files/11075404/MobaXterm_192.168.1.51_20230327_083743.txt)

**Comment by ilike2burnthing** [comment]
You didn't redact the second instance of your IP on your gist.

Are Prowlarr and FlareSolverr using 2 different IPs?

**Reply by SadE54** [reply]
Yeah they're using different IPs. I guess it's the problem, and that's why this morning I tested quickly with a jackett container on the same host than flaresolverr and it's working. The error message was somehow misleading .

**Reply by ilike2burnthing** [reply]
The challenge wasn't detected, no issue there, but the returned cookie would be invalid as it's for a different IP.
